pkg/strategy/controller.go

```diff
@@ -231,13 +231,18 @@ func enableLogOut(uiOpts map[string]interface{}, spec *v1.JaegerSpec) {
 		return
 	}
 
+	docURL := viper.GetString("documentation-url")
+	if docURL == "" {
+		docURL = "https://www.jaegertracing.io/docs/latest"
```

Comment:
This shouldn't be necessary, as you should get the default value from viper.

Reply:
Yeah thought that - but when running the unit test from within vscode it failed.

Reply:
The default value is set in the command, so, it's not being set at all during the unit tests. You'll need something similar to this:

https://github.com/jaegertracing/jaeger-operator/blob/e377c2bc3a5008bd573ad177ff5f6036af2f3649/pkg/deployment/agent_test.go#L17-L23

Reply:
Ok thanks.